File: enchantgui/api/src/main/java/cc/mewcraft/enchantment/gui/api/UiEnchantPlugin.java
package cc.mewcraft.enchantment.gui.api;

import cc.mewcraft.mewcore.message.Translations;
import cc.mewcraft.mewcore.plugin.MeowJavaPlugin;
import com.google.inject.Injector;

public abstract class UiEnchantPlugin extends MeowJavaPlugin {
    public abstract Translations getLang();

    public abstract Injector getInjector();
}

File: enchantgui/api/src/main/java/cc/mewcraft/enchantment/gui/api/UiEnchantPlugin.kt
package cc.mewcraft.enchantment.gui.api

import cc.mewcraft.mewcore.message.Translations
import cc.mewcraft.mewcore.plugin.MeowJavaPlugin
import com.google.inject.Injector

abstract class UiEnchantPlugin : MeowJavaPlugin() {
    lateinit var languages: Translations
    lateinit var injector: Injector
}
